_raml/apps/carts/examples/post_carts_cartid_multilineitem_request.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "productId": "54E72E1F-746A-8B6E-8E41-0A0C05E64669",
+    "quantity": 1
+  },
+  {
+    "productId": "54E72E1F-746A-8B6E-8E41-0A0C05E62569",
+    "quantity": 1
```

Comment:
Please use different GUIDs and other quantities than 1 for a better example and probably 3 items.